visidata/loaders/xlsx.py

```diff
@@ -53,16 +53,39 @@ def iterload(self):
         for rownum in Progress(range(worksheet.nrows)):
             yield list(worksheet.cell(rownum, colnum).value for colnum in range(worksheet.ncols))
 
-            
+
+def excel_safename(name, used_names):
+    'convert sheet names to comply with Excel standards'
+
+    # name does not exceed 31 characters
+    excel_limit = 31
+    if len(name) > excel_limit:
+        short_name = name[:excel_limit]
+        used_names.append(short_name)
+        suffix = '_' + str(used_names.count(short_name))
+        name = short_name[:-len(suffix)] + suffix
+
+    # name does not contain any of the following characters:  :  \  /  ?  *  [  or  ]
+    name = ''.join([ '_' if c in '\\/?*[]' else c for c in name])
+
+    # strip leading and trailing _
+    name.strip("_")
```

Comment:
strings are immutable in python, so this should be `name = name.strip("_")`

Reply:
Done!